src/auth/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const jwtConstants = {
```

Comment:
in my opinion this should not be in the repository. It is sensitive data, so I would put in in config file